libmscore/note.cpp

```diff
@@ -2061,6 +2061,34 @@ void Note::layout()
                   if (chord()->measure() != tieBack()->startNote()->chord()->measure() || el().size() > 0)
                         paren = true;
                   }
+
+            if (auto scoreElement = links()->mainElement()) {
+                  if (scoreElement->isNote())  {
+                        if (auto n = toNote(scoreElement)) {
+                              int stringNoFromText = _string;
```

Comment:
On reading I stumble accross this, could ypou change to `stringNrFromText` please?

Reply:
Me too earlier ( was following the convention of measures: )
![image](https://github.com/Jojo-Schmitz/MuseScore/assets/7139517/de18bd88-194c-4ceb-85ee-638d97eb5a78).

I'll change it to whichString. Good catch on the mtest thing -  apparently `links()` aren't guaranteed to be 'prepared'